tests/php/maybe-generate-wp-cli-coverage.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use SebastianBergmann\CodeCoverage\Filter;
+use SebastianBergmann\CodeCoverage\Driver\Selector;
+use SebastianBergmann\CodeCoverage\CodeCoverage;
+use SebastianBergmann\CodeCoverage\Report\Clover;
+
+$root_folder = realpath( dirname( dirname( __DIR__ ) ) );
+
+if ( ! class_exists( 'SebastianBergmann\CodeCoverage\Filter' ) ) {
+	require "{$root_folder}/vendor/autoload.php";
```

Comment:
```suggestion
	require_once "{$root_folder}/vendor/autoload.php";
```

Reply:
As we already check whether the class exists, the `require_once` is only slower than `require` in the case of the expected behavior, and might cloak a problem if the dependency is actually missing. In that case, I'd prefer for the code to fail right away in here to show that class was missing even though the autoloader was already required before.